astropy/constants/constant.py

```diff
@@ -85,6 +85,30 @@ class Constant(Quantity, metaclass=ConstantMeta):
 
     def __new__(cls, abbrev, name, value, unit, uncertainty,
                 reference=None, system=None):
+
+        """
```

Comment:
Good to add this but perhaps just add to the class docstring.

Reply:
👍 done